backend/src/preprocessing.py

```diff
@@ -148,25 +151,39 @@ def registration(packet):
     [in] packet : pipeline packet
     [post] images registered in place
     """
-    return  # TODO remove
-    # TODO how are these gonna be and what to return
-    reference_color, align_color = packet
+    subject = packet.get_subject()
+    align_color = subject[1]
+    reference_color = subject[0]
 
     # grayscale
-    img1 = cvtColor(reference_color, COLOR_BGR2GRAY)
-    img2 = cvtColor(align_color, COLOR_BGR2GRAY)
-    height, width = img2.shape
-
-    # create ORB detector
-    orb_detector = ORB_create(5000)
-    key_points1, descriptors1 = orb_detector.detectAndCompute(img1, None)
-    key_points2, descriptors2 = orb_detector.detectAndCompute(img2, None)
+    img1_gray = cvtColor(align_color, COLOR_RGB2GRAY)
+    img2_gray = cvtColor(reference_color, COLOR_RGB2GRAY)
+    height, width = img2_gray.shape
+
+    detector = SIFT_create(2000)
+    descriptor = SIFT_create(2000)
+
+    img1_gray_norm = normalize(img1_gray, None, 0, 255, NORM_MINMAX).astype('uint8')
+    img2_gray_norm = normalize(img2_gray, None, 0, 255, NORM_MINMAX).astype('uint8')
```

Comment:
For memory saving please keep it in the same array or delete the img(n)_grey arrays after